### PR TITLE
RESTEASY-2161 remove @NoJackson annotation #1869 (rebased)

### PR DESCRIPTION
**Pull request by liweinan**
rebased version of: https://github.com/resteasy/Resteasy/pull/1869

### CODE REVIEW

File: docbook/reference/en/en-US/modules/Jaxb.xml [hunk 1]
```diff
@@ -390,209 +376,6 @@ public class Book
...
-      <para>
-
-<programlisting>
-{ "book" :
-    {
-        "@title":"EJB 3.0",
-        "author":"Bill Burke",
-        "ISBN":596529260
-    }
-}
```
Comment (ronsigal) [inline docbook/reference/en/en-US/modules/Jaxb.xml]: Hey @liweinan,

Having removed the Jettison example, maybe you could include something like "The JSON output produced by Jackson would look like this:" with a Book in JSON format.

Comment (liweinan) [reply docbook/reference/en/en-US/modules/Jaxb.xml]: yes sir!

Comment (ronsigal) [reply docbook/reference/en/en-US/modules/Jaxb.xml]: Lol.

**Comment by ronsigal** [comment]
Ok, looks good. Thanks, Wei Nan!!

**Pull request merged**